newIDE/app/src/InstancesEditor/SelectedInstances.js

```diff
@@ -139,14 +174,44 @@ export default class SelectedInstances {
     objectButton: PIXI.Graphics,
     onMove: (event: PanMoveEvent) => void,
     onEnd: () => void,
+    onPanMove: (event: PanMoveEvent) => void,
+    onPanEnd: () => void,
     cursor: string
   ) {
     objectButton.eventMode = 'static';
     objectButton.buttonMode = true;
     objectButton.cursor = cursor;
     panable(objectButton);
-    objectButton.addEventListener('panmove', onMove);
-    objectButton.addEventListener('panend', onEnd);
+
+    // change cursor style if space is pressed
+    objectButton.addEventListener('mousemove', event => {
+      if (this.keyboardShortcuts.shouldMoveView()) {
+        objectButton.cursor = 'grab';
+      } else {
+        objectButton.cursor = cursor;
+      }
+    });
+
+    let isPanMoveEvent = false;
```

Comment:
Could you rename this boolean with something more descriptive, like "shouldPanView" or "shouldMoveView"?

Reply:
Just to clarify, you are suggesting using the shouldMoveView instead of the boolean?
If so, it would call the resizeEnd instead of the panEnd if the user released the spacebar first.